contracts/actions/flashloan/FLAaveV3CarryDebt.sol

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../ActionBase.sol";
+import "../../interfaces/IDSProxy.sol";
+import "../../interfaces/IFLParamGetter.sol";
+import "../../interfaces/aaveV3/IPoolV3.sol";
+import "../../core/strategy/StrategyModel.sol";
+import "../../utils/TokenUtils.sol";
+import "../../utils/ReentrancyGuard.sol";
+import "./helpers/FLHelper.sol";
+import "../../interfaces/flashloan/IFlashLoanBase.sol";
+import "../../interfaces/aaveV3/IDebtToken.sol";
+
+/// @title Action that gets and receives a FL from Aave V3 and does not return funds but opens debt position on Aave V3
```

Comment:
Write a ///@dev comment explaining this action, that for it to work credit delegation needs to be given, but the exact amount and why the check in the action for it to not be left over